frontend/src/metabase-types/store/qb.ts

```diff
@@ -37,6 +39,8 @@ export interface QueryBuilderUIControls {
   isShowingNotebookNativePreview: boolean;
   notebookNativePreviewSidebarWidth: number | null;
   showSidebarTitle: boolean;
+  modal: QueryModalType | null;
+  modalContext: TimelineEventId | null;
```

Comment:
These attributes are currently undefined in `DEFAULT_UI_CONTROLS` (initial state of the reducer), let's add them there.

Reply:
Way ahead of you 😄 https://github.com/metabase/metabase/pull/49276/files

Reply:
weird, I don't see it there :)

Reply:
I thought I could change the branch faster 😂 should be there now